src/main/index.js

```diff
@@ -225,6 +225,40 @@ function runApp() {
     }
   })
 
+  // If in windows, add Windows Jump List task 'New Window'
+  if (process.platform === 'win32') {
+    app.setJumpList([{
+      type: 'tasks',
+      items: [{
+        type: 'task',
+        program: process.execPath,
+        args: '--new-window',
+        title: 'New Window',
+        description: 'Open New Window',
+        iconPath: process.execPath,
+        iconIndex: 0
+      }]
+    }])
+  }
+
+  // If in Mac, add macOS Dock menu with 'New Window' option
+  if (process.platform === 'darwin') {
+    const dockMenu = Menu.buildFromTemplate([
+      {
+        label: 'New Window',
+        click: () => {
+          createWindow({
+            replaceMainWindow: false,
+            showWindowNow: true
+          })
+        }
+      }
+    ])
+    app.whenReady().then(() => {
+      app.dock.setMenu(dockMenu)
+    })
+  }
```

Comment:
Please move this inside the existing event handler for the `ready` event instead of adding another one.